code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -273,10 +273,12 @@
 	weight = 2
 	cost = 20
 	requirements = list(90,90,90,80,60,40,30,20,10,10)
+	ruleset_lazy_templates = list(LAZY_TEMPLATE_KEY_WIZARDDEN)
 
-/datum/dynamic_ruleset/roundstart/wizard/acceptable(population=0, threat=0)
-	SSmapping.lazy_load_template(LAZY_TEMPLATE_KEY_WIZARDDEN)
-	if(GLOB.wizardstart.len == 0)
+/datum/dynamic_ruleset/roundstart/wizard/ready(forced = FALSE)
+	if(!check_candidates())
```

Comment:
I'm still not sure we should be changing how the rulesets act in this PR, old Dynamic code is pretty clunky and I wouldn't be surprised if this broke something

Reply:
Just made it copy the other ruleset logic